Formula/swift-format.rb

```diff
@@ -2,8 +2,8 @@ class SwiftFormat < Formula
   desc "Formatting technology for Swift source code"
   homepage "https://github.com/apple/swift-format"
   url "https://github.com/apple/swift-format.git",
-      tag:      "0.50700.0",
-      revision: "3dd9b517b9e9846435aa782d769ef5825e7c2d65"
+      tag:      "0.50700.1",
+      revision: "5f184220d032a019a63df457cdea4b9c8241e911"
   license "Apache-2.0"
   version_scheme 1
   head "https://github.com/apple/swift-format.git", branch: "main"
```

Comment:
can you please change  
**depends_on xcode: ["14.0", :build**] 
to  
**depends_on xcode: ["14.1", :build]**

Reply:
Done. However I believe maintainers wanted to update it along the last swift update.

Reply:
> However I believe maintainers wanted to update it along the last swift update.

As above, turns out Xcode 14.1 isn't actually compatible with this release. So decided to go for a partial update for Linux only and keep this open to track an eventual macOS update.